server/middleware/github-griller/github.comments.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+var Promise = require('bluebird');
+var parser = require('./github.parse');
+
+exports.getAll = function(req, repo, dep) {
+  console.log("mm-----in the get all comments")
+  this.client = req.user.githubAccess;
+  console.log('0Little farther down')
+  this.githubFunc = Promise.promisify(this.client.issues.repoComments);
```

Comment:
@ssarda77 by attaching the github client function to 'this.githubFunc', we're able to use a generic function 'getRemainingPages'.  'getRemainingPages' will call whatever the githubFunc is repeatedly until all the pages for issues or repos or whatever are gotten.